### PR TITLE
fix: Get rid of `$crate` in expansions shown to the user

### PR DESCRIPTION
**Pull request by ChayimFriedman2**
Be it "Expand Macro Recursively", "Inline macro" or few other things.

We replace it with the crate name, as should've always been.

Probably fixes some issues, but I don't know what they are.

### CODE REVIEW

File: crates/syntax-bridge/src/prettify_macro_expansion.rs [hunk 1]
```diff
@@ -8,10 +8,18 @@ use syntax::{
 };
 
 /// Renders a [`SyntaxNode`] with whitespace inserted between tokens that require them.
-pub fn insert_ws_into(syn: SyntaxNode) -> SyntaxNode {
+///
+/// This is an internal API that is only exported because `mbe` needs it for tests and cannot depend
+/// on `hir-expand`. For any purpose other than tests, you are supposed to use the `prettify_macro_expansion`
+/// from `hir-expand` that handles `$crate` for you.
+pub fn prettify_macro_expansion(
```
Comment (Veykril) [inline crates/syntax-bridge/src/prettify_macro_expansion.rs:15]: A trick we use for other similar "don't use" APIs is putting a `#[deprecrated = "msg"]` on it and have it allowed on the few callers

**Review by Veykril — APPROVED** [review]

**Comment by Veykril** [comment]
Thanks!
@bors d+
Feel free to merge as is or with the deprecation idea applied

**Comment by ChayimFriedman2** [comment]
@Veykril `@bors d+` does not delegate...

**Comment by Veykril** [comment]
ugh I'll never get used to having to type out delegate...
@bors r+

**Pull request merged**